packages/yew/tests/suspense.rs

```diff
@@ -119,6 +119,8 @@ async fn suspense_works() {
         .unwrap()
         .click();
 
+    TimeoutFuture::new(0).await;
```

Comment:
Kinda odd that this is the only place which uses `TimeoutFuture` instead of `sleep`. Not saying that you need to change it tho

Reply:
This test is written before sleep is available and uses sleep to delay. I am just too lazy to change it to sleep.